modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -6,6 +6,9 @@ label:AuraDB-Virtual-Dedicated-Cloud[]
 label:AuraDS-Enterprise[]
 label:AuraDB-Business-Critical[]
 
+* AuraDB Virtual Dedicated Cloud supports Organization SSO and Project SSO.
```

Comment:
hey @KingJohanna 😊 is Organization SSO and Project SSO available for AuraDS-Enterprise?